test/integration/tests/snapshot-prefetch.test.ts

```diff
@@ -0,0 +1,153 @@
+import {PolarisServer} from '../../../src';
+import {initializeDatabase} from '../server/dal/data-initalizer';
+import {startTestServer, stopTestServer} from '../server/test-server';
+import {graphqlRawRequest} from '../server/utils/graphql-client';
+import {snapshotRequest} from '../server/utils/snapshot-client';
+import * as paginatedQuery from './jsonRequestsAndHeaders/paginatedQuery.json';
+
+let polarisServer: PolarisServer;
+
+afterEach(async () => {
+    await stopTestServer(polarisServer);
+});
+
+describe('snapshot pagination tests with auto disabled', () => {
+    describe('snap request is true', () => {
+        describe('prefetch is 1', () => {
```

Comment:
Create beforeEach for this specific describe suite, and init the server and the db.

Reply:
will not work.